core/scripts/cli/sponsor/redeem.js

```diff
@@ -22,15 +28,15 @@ const redeem = async (web3, artifacts, emp) => {
     .div(toBN(position.tokensOutstanding.toString()));
```

Comment:
For consistency should you be using BigNumber.js here? I think the rounding works in your favor here by rounding down, but prob worth being safer

Reply:
Prob outside the scope of this PR

Reply:
Like you said, I think we're safe in this case, but we should make sure there aren't other places in the CLI where you'll be off by 1 wei and then fail transactions.